graylog2-server/src/main/java/org/graylog2/plugin/inputs/transports/ThrottleableTransport.java

```diff
@@ -212,4 +227,23 @@ public void blockUntilUnthrottled() {
             // ignore
         }
     }
+
+    /**
+     * Blocks until the blockLatch is released or until timeout is exceeded.
+     *
+     * @param timeout the maximum time to wait
+     * @param unit    the time unit of the {@code timeout} argument
```

Comment:
Please document the return value in here.

Reply:
Great catch. Done.